utils/src/main/java/org/apache/cloudstack/utils/imagestore/ImageStoreUtil.java

```diff
@@ -81,7 +81,7 @@ public static String checkTemplateFormat(String path, String uripath) {
             return "";
         }
 
-        if (output.contains("ISO 9660") && isCorrectExtension(uripath, "iso")) {
+        if ((output.startsWith("ISO 9660") || output.startsWith("DOS/MBR")) && isCorrectExtension(uripath, "iso")) {
```

Comment:
Well, the *file* command simply behaves differently under Debian 9 then it does under Debian 7.

See my commit for this change.